### PR TITLE
VOXY-1444 Pytho2.7/3.7 compatibility

### PR DESCRIPTION
**Pull request by miphreal**
Resolves JIRA [VOXY-1444](https://voxy.atlassian.net/browse/VOXY-1444)

## Changes
- add an explicit check against py37
- fix some absolute imports
- import `unicode_literals` in all files, so it'll be easier to catch unicode errors
- update docs and Makefile

## Post merge actions

- [x] bump the version from `1.0.2` to `1.1.0` (@miphreal)

### CODE REVIEW

**Comment by miphreal** [comment]
@carlnegroatvoxy, @icaroseara 
this PR is ready for review, please take a look when you have time

File: setup.py [hunk 1]
```diff
@@ -3,16 +3,24 @@
...
 _version_re = re.compile(r'__version__\s+=\s+(.*)')
 
 here = path.abspath(path.dirname(__file__))
 
 with open(path.join(here, 'okta', '__init__.py'), 'rb') as f:
+    # `version` has "vX.Y.Z" format
     version = str(ast.literal_eval(_version_re.search(
         f.read().decode('utf-8')).group(1)))
 
+if sys.argv[-1] == 'push-and-tag':
```
Comment (miphreal) [inline setup.py:19]: added this logic so it's consistent with other internal packages

**Comment by icaroseara** [comment]
@carlnegroatvoxy, could you review this PR?

**Comment by icaroseara** [comment]
@carlnegroatvoxy , could you review this PR?

**Review by carlnegroatvoxy — APPROVED** [review]
Looks good and tests pass 👍

**Pull request merged**